subprojects/core/src/test/groovy/org/gradle/api/internal/project/taskfactory/AnnotationProcessingTaskFactoryTest.groovy

```diff
@@ -41,8 +40,7 @@ class AnnotationProcessingTaskFactoryTest extends AbstractProjectBuilderSpec {
     private AnnotationProcessingTaskFactory factory
     private ITaskFactory delegate
     private TaskClassInfoStore taskClassInfoStore
-    private classImplementationHasher = Mock(ClassImplementationHasher)
-    def propertyWalker = new DefaultPropertyWalker(new DefaultPropertyMetadataStore([]), classImplementationHasher)
+    def propertyWalker = new DefaultPropertyWalker(new DefaultPropertyMetadataStore([]))
```

Comment:
This is so much more correct this way. I mean: the walker should have no knowledge of the implementation hasher.